spdx-utils/src/main/kotlin/model/SpdxFile.kt

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.spdx.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.ossreviewtoolkit.spdx.isSpdxExpressionOrNotPresent
+
+/**
+ * Provides important meta-data about a particular file of a software package, see also
+ * https://github.com/spdx/spdx-spec/blob/master/chapters/4-file-information.md.
+ */
+@JsonIgnoreProperties("ranges") // TODO: Implement ranges which is broken in the specification examples.
+data class SpdxFile(
+    /**
+     * A uniquely identifies this [SpdxFile] within a SPDX document.
```

Comment:
Remove "A", or "A unique identifier for".

Reply:
Done.